lib/gems/pending/appliance_console.rb

```diff
@@ -307,7 +307,7 @@ module ApplianceConsole
             system_hosts.hostname = new_host
             system_hosts.set_loopback_hostname(new_host)
             system_hosts.save
-            LinuxAdmin::Service.new("network").restart
+            LinuxAdmin::Service.new("NetworkManager").restart
```

Comment:
In my search, some documents said we need to restart the machine, while some docs require to at least restart network service to make hostname change take effect: https://www.howtogeek.com/50631/how-to-change-your-linux-hostname-without-rebooting/